### DISCUSSION TITLE
Connecting ApicurioStudio to Postgresql

### DISCUSSION THREAD

**Discussion by SkyTaill** [ApicurioArchive/apicurio-studio#2217] [Q&A]
Hi,
I have some problem with apicurio-api service - some problem with connect to Postgresql. what could be the problem? 
 - postgres:15.2 
 - apicurio-studio-platforms-quarkus-api 0.2.62

2023-06-20 10:30:53,587 ERROR [io.und.req.io] (executor-thread-0) Exception handling request 9378c0fb-3099-4e19-b518-c798b60ac8cc-1 to /currentuser/activity: org.jboss.resteasy.spi.UnhandledException: org.jdbi.v3.core.statement.UnableToExecuteStatementException: org.postgresql.util.PSQLException: ERROR: syntax error at or near "AUTO_INCREMENT"
  Position: 37 [statement:"CREATE TABLE api_designs (id BIGINT AUTO_INCREMENT NOT NULL, name VARCHAR(255) NOT NULL, description VARCHAR(1024), created_by VARCHAR(255) NOT NULL, created_on TIMESTAMP NOT NULL, tags VARCHAR(2048), api_type VARCHAR(255) NOT NULL)", arguments:{positional:{}, named:{}, finder:[]}]

<img width="868" alt="image" src="https://github.com/Apicurio/apicurio-studio/assets/28933342/621aa773-3042-4e6d-9e71-25f4d91da022">

**Comment by andreaTP** [comment]
Seems like the Database migration executed is not the one specified for postgres ... can you share more about your setup/env variable etc. etc.?

**Reply by SkyTaill** [reply]
name: apicurio-api
           env:
            - name: APICURIO_DB_CONNECTION_URL
              value: >-
                jdbc:postgresql://my-apicurio-postgresql.apicurio.svc.cluster.local:5432/postgres
            - name: APICURIO_DB_DRIVER_NAME
              value: postgresql
            - name: APICURIO_DB_INITIALIZE
              value: 'true'
            - name: APICURIO_HUB_STORAGE_JDBC_INIT
              value: 'true'
            - name: APICURIO_DB_TYPE
              value: postgresql9
            - name: APICURIO_DB_USER_NAME
              value: postgres
            - name: APICURIO_MICROCKS_API_URL
              value: 'https://apicurio-ws-https.apps.ocp.syn-dev.solution.sbt/api'
            - name: APICURIO_MICROCKS_CLIENT_ID
              value: microcks-serviceaccount
            - name: APICURIO_KC_AUTH_URL
              value: 'https://apicurio-kc.apps.ocp.syn-dev.solution.sbt/auth'
            - name: APICURIO_KC_REALM
              value: apicurio
            - name: APICURIO_KC_CLIENT_ID
              value: apicurio
            - name: APICURIO_KC_CLIENT_SECRET
              value: pass
            - name: APICURIO_MICROCKS_CLIENT_SECRET
              value: apicuriomr
            - name: APICURIO_SHARE_FOR_EVERYONE
              value: 'true'
            - name: APICURIO_LOGGING_LEVEL
              value: debug
            - name: APICURIO_DB_PASSWORD
              value: Apz36fmPsp
            - name: APICURIO_GITLAB_API_URL
              value: 'https://api.github.com'
            - name: JAVA_TOOL_OPTIONS
              value: >-
                -Dquarkus.http.cors=true
                -Dquarkus.http.cors.methods=GET,PUT,POST,DELETE,PATCH,OPTIONS
                -Dquarkus.http.cors.headers=accept,origin,authorization,content-type,x-requested-with
                -Dquarkus.http.cors.exposed-headers=Content-Disposition
                -Dquarkus.http.cors.access-control-max-age=24H
                -Djava.net.preferIPv4Stack=true
                -Djavax.net.ssl.trustStore=/trust/truststore.jks
                -Djavax.net.ssl.trustStorePassword=secret
                -Dquarkus.http.cors=true  
                -Dapicurio.hub.github.baseUrl=https://github.com
                -Dapicurio.hub.github.apiUrl=https://api.github.com

         name: postgresql
          env:
            - name: BITNAMI_DEBUG
              value: 'false'
            - name: POSTGRESQL_PORT_NUMBER
              value: '5432'
            - name: POSTGRESQL_VOLUME_DIR
              value: /var/lib/postgresql
            - name: PGDATA
              value: /var/lib/postgresql/data
            - name: POSTGRES_PASSWORD
              valueFrom:
                secretKeyRef:
                  name: my-apicurio-postgresql
                  key: postgres-password
            - name: POSTGRESQL_ENABLE_LDAP
              value: 'no'
            - name: POSTGRESQL_ENABLE_TLS
              value: 'no'
            - name: POSTGRESQL_LOG_HOSTNAME
              value: 'false'
            - name: POSTGRESQL_LOG_CONNECTIONS
              value: 'false'
            - name: POSTGRESQL_LOG_DISCONNECTIONS
              value: 'false'
            - name: POSTGRESQL_PGAUDIT_LOG_CATALOG
              value: 'off'
            - name: POSTGRESQL_CLIENT_MIN_MESSAGES
              value: error
            - name: POSTGRESQL_SHARED_PRELOAD_LIBRARIES
              value: pgaudit

**Reply by andreaTP** [reply]
Needs more investigation, opened #2218

**Reply by EricWittmann** [reply]
I meant to look into this last week but haven't had time yet.  But it's next on my list!

**Comment by SkyTaill** [comment]
Hi!
I found what was the problem. I specified the wrong env
